components/District.tsx

```diff
@@ -1,13 +1,18 @@
 type props = {
+  province: number;
   shape: string;
 };
 
-export default function District({ shape }: props) {
+const colors: string[] = Array(7)
+  .fill('hsl(_)')
+  .map((c) => c.replace(/_/g, `${Math.floor(Math.random()*360)}, 100%, 50%`));
```

Comment:
Can you extract this to a utility function file?

Reply:
Maybe this is better?
```ts
const colors: string[] = Array(7)
  .map((_) => Math.floor(Math.random() * 360))
  .map((c) => `hsl(${c}, 100%, 50%)`);
```

Reply:
> Maybe this is better?
> 
> ```ts
> const colors: string[] = Array(7)
>   .map((_) => Math.floor(Math.random() * 360))
>   .map((c) => `hsl(${c}, 100%, 50%)`);
> ```

This doesn't work apparently sorry